language_service/src/state/tests.rs

```diff
@@ -152,6 +160,7 @@ async fn close_last_doc_in_openqasm_project() {
             "openqasm_files/self-contained.qasm",
             1,
             "include \"stdgates.inc\";\nqubit q;\nreset q;\nx q;\nh q;\nbit c = measure q;\n",
+            "openqasm",
```

Comment:
Is it worth mimicking a few more of the tests in this file for document updates, completions, and error reporting with openqasm docs?

Reply:
I was going to add more when I got include file loading implemented.